### PR TITLE
Add usage and update instructions to Readme.md

### LINKED ISSUE #86: Add usage and update instructions to Readme.md
- add usage instructions
- add package update/release instructions
- deploy Storybook to gh-pages

### PR DESCRIPTION
**Pull request by JiwoonKim**
Closes #86 
Add instructions for usage, contribution, publishing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +46,16 @@ const customTheme = createMuiTheme({
...
+This repository uses Neutrino and `neutrino-preset-react-components` for developing, previewing, and building React components. To get started:
+- Fork and clone this repo.
+- Install the dependencies with `yarn`.
+- Start the development server with `yarn start` (use `CTRL-C` to exit).
+
+Feel free to open an issue, submit a pull request, or contribute however you would like. Understand that this documentation is still a work in progress, so file an issue or submit a PR to ask questions or make improvements. Thanks!
+
+## Publishing
+- Bump the version of the package with `npm version {version-name}` (patch, minor, major, etc.).
+- Gererate the compiled component for publishing to npm with `yarn build`.
```
Comment (helfi92) [inline README.md]: `s/Gererate/Generate/`

File: README.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ a viewer component that displays JSON schemas (built using [material-ui](https:/
...
+```js
+import SchemaViewer from 'material-ui-json-schema-viewer';
+
+<SchemaViewer schema={jsonSchema} references={schemaReferences} />
+```
+
+| Prop | Type | Required | Description |
+|:---|:---|:---|:---|
+| `schema` | Object | ✓ | A JSON schema object. |
+| `references` | Array| ✓ | An array of JSON schema objects (including the schema to render), which the schema may refer to. (each schema within the references must include an `$id` property to use for key-value mapping the references) |
```
Comment (helfi92) [inline README.md]: Is `references` required even if the schema does not include a ref?

File: README.md [hunk 3]
```diff
@@ -29,3 +46,16 @@ const customTheme = createMuiTheme({
     <SchemaViewer />
 </ThemeProvider>
 ```
+
+## Contributing
+This repository uses Neutrino and `neutrino-preset-react-components` for developing, previewing, and building React components. To get started:
```
Comment (helfi92) [inline README.md]: `neutrino-preset-react-components` is not used in this repo. We can omit this:

This repository uses Neutrino for developing, and building React components. To get started:

File: README.md [hunk 4]
```diff
@@ -29,3 +46,16 @@ const customTheme = createMuiTheme({
...
+- Fork and clone this repo.
+- Install the dependencies with `yarn`.
+- Start the development server with `yarn start` (use `CTRL-C` to exit).
```
Comment (helfi92) [inline README.md]: Maybe we should have yarn start run the storybook rather and remove the storybook script in package.json.

File: README.md (hunk 1 above)
Comment (JiwoonKim) [reply README.md]: Applied changes!

File: README.md (hunk 2 above)
Comment (JiwoonKim) [reply README.md]: `references` is not required when there is no ref. Fixed this part!

File: README.md (hunk 3 above)
Comment (JiwoonKim) [reply README.md]: Applied changes!

File: README.md (hunk 4 above)
Comment (JiwoonKim) [reply README.md]: Applied changes!

**Pull request merged**